doc/prosoul-grimoirelab.md

```diff
@@ -1,39 +1,144 @@
+- Elasticsearch at https://localhost:9200/
+- Kibiter at http://localhost:5601/
+- Prosoul at http://localhost:8000/
 
-You need a GitHub API token in order to load GitHub data from GrimoireLab projects.
+Let's perform an assessment for a few grimoirelab projects using the `gitqm` and `githubqm` backends and the Developer Quality Model (for GrimoireLab).
+ 
+### Load GrimoireLab Data
 
-With the `prosoul/tests/grimoirelab/init.sh` script the data for projects is collected,
-and the quality model for doing the assessment and visualization is loaded.
+You need to setup [SirMordred](https://github.com/chaoss/grimoirelab-sirmordred) project. You can use the [Getting-Started.md](https://github.com/chaoss/grimoirelab-sirmordred/blob/master/Getting-Started.md#getting-started-) guide and omit the `Getting the containers` section as the Elasticsearch and Kibiter are already available.
 
+Now that you have the project configured in the PyCharm, we can execute micro-mordred/sirmordred. All you need to define the setup.cfg and projects.json with the required configurations and execute the backends.
+
+Let's use micro-mordred for this purpose.
+
+projects.json
 ```
-acs@~/devel/prosoul/tests/grimoirelab $ ./init.sh <github-api-token>
+{
+    "GrimoireLab": {
+        "gitqm": [
+            "https://github.com/chaoss/grimoirelab-elk",
+            "https://github.com/chaoss/grimoirelab-kidash",
+            "https://github.com/chaoss/grimoirelab-sirmordred",
+            "https://github.com/chaoss/grimoirelab-perceval",
+            "https://github.com/chaoss/grimoirelab-sortinghat"
+        ],
+        "githubqm:issue": [
+            "https://github.com/chaoss/grimoirelab-elk",
+            "https://github.com/chaoss/grimoirelab-kidash",
+            "https://github.com/chaoss/grimoirelab-sirmordred",
+            "https://github.com/chaoss/grimoirelab-perceval",
+            "https://github.com/chaoss/grimoirelab-sortinghat"
+        ]
+}
 ```
 
-Once the script execution has finished, you have in Elasticsearch the projects data needed for
-doing the assessment and visualization of them using ProSoul.
+setup.cfg
+```
+[gitqm]
+raw_index = grimoirelab_gitqm_raw
+enriched_index = grimoirelab_gitqm_enriched
+category = commit
+
+[githubqm:issue]
+raw_index = grimoirelab_githubqm_issues_raw
+enriched_index = grimoirelab_githubqm_issues_enriched
+api-token = xxxx
+sleep-for-rate = true
+sleep-time = 300
+category = issue
+no-archive = true
+```
+You can find all the configurations here, link.
+
+Now, execute the raw and enrich tasks of micro-mordred.
+```
+micro.py --raw --enrich --cfg ./setup.cfg --backends gitqm githubqm:issue
+```
+
+Setup alias for all the enriched indices.
+```
+POST /_aliases
+{
+  "actions": [
+    {
+      "add": {
+        "indices": [
+          "grimoirelab_gitqm_enriched",
+          "grimoirelab_githubqm_issues_enriched"
+        ],
+        "alias": "all_qm_data"
+      }
+    }
+  ]
+}
+```
```

Comment:
Would it be good if we can automate the alias adding?

I understand that we can add an alias to the enriched indices using sirmordred by having an entry in the [aliases.json](https://github.com/chaoss/grimoirelab-sirmordred/blob/master/aliases.json).

So, we can add a common alias (`all_qm_data`) to all the enriched QM enriched indices.

The only disadvantage I see is all the enriched indices will be a part of the assessment.
If we enrich `githubqm:pull` but we don't want it in the assessment, then we have to manually remove the alias.